gatsby-node.js

```diff
@@ -143,13 +143,12 @@ const createTagPageType = async (createPage, pageMetadata, stitchType) => {
         if (!name) return null;
         else {
             const urlSuffix = isAuthor
-                ? encodeURIComponent(
-                      name
-                          .toLowerCase()
-                          .split(' ')
-                          .join('-')
-                  )
-                : encodeURIComponent(name.toLowerCase());
+                ? name
+                      .toLowerCase()
+                      .split(' ')
+                      .join('-')
+                      .replace(/\W/g, '-')
```

Comment:
What about writing a helper function with this regex and we can wrap all the necessary strings in this pr with that? 

Also curious why we do a split and join here when the regex should apply to spaces anyway?

Reply:
Yeah we can simplify this a little bit here.